readme.md

```diff
@@ -351,7 +351,7 @@ GitHub implemented dozens of features that used to be part of Refined GitHub 
 - [Blog post](https://github.blog/changelog/2019-02-14-prompt-to-clean-up-merged-forks): [Quickly delete a forked repo after its pull request has been merged.](https://cloud.githubusercontent.com/assets/170270/13520281/b2c9335c-e211-11e5-9e36-b0f325166356.png)
 - [Blog post](https://github.blog/changelog/2019-03-05-exclude-labels-from-search/): [Exclude PR/issue filters from their list](https://user-images.githubusercontent.com/1402241/48470535-493cfb00-e824-11e8-863a-964f52b62553.png) with <kbd>alt</kbd> <kbd>click</kbd>.
 - [Blog post](https://github.blog/changelog/2019-03-18-sticky-file-headers-in-pull-requests/): [File headers are always visible.](https://user-images.githubusercontent.com/81981/28682784-78bac340-72fe-11e7-9386-bdbab7703693.gif)
-- [Blog Post](https://github.blog/changelog/2019-03-29-linking-branch-names-on-pull-request-headers/): [Linkify branch names in pull requets.](https://github.com/sindresorhus/refined-github/issues/1)
+- [Blog Post](https://github.blog/changelog/2019-03-29-linking-branch-names-on-pull-request-headers/): [Linkify branch names in pull requests.](https://github.com/sindresorhus/refined-github/issues/1)
```

Comment:
This can be merged, but recently we discussed we’re gonna drop this section and replace it with an “closed PR search” link

Reply:
Should I leave this section as is?

(And I'm not able to find the discussion 😅)

Reply:
> (And I'm not able to find the discussion)

@kidonng: https://github.com/sindresorhus/refined-github/pull/3777#discussion_r532124576
Might as well do it in this PR and rename it "Lint readme" or something.

Reply:
Thanks for the information! Will check it out later.